.github/workflows/pipeline-deploy-app-develop.yaml

```diff
@@ -66,20 +66,55 @@ jobs:
           unit-tests,
           validate-terraform,
         ]
-      uses: ./.github/workflows/deploy-infrastructure-stack.yaml
+      uses: ./.github/workflows/action-infrastructure-stack.yaml
       with:
         github_environment: dev
         stack: capacity_management
```

Comment:
also not for now maybe but we did talk about renaming capacity_management stack as application - not sure how straightforward that is or if a search and replace would cover it

Reply:
Agreed to do this during migration to DoS repos